packages/swingset-runner/demo/encouragementBot/vat-user.js

```diff
@@ -12,7 +11,7 @@ export function buildRootObject(_vatPowers) {
         .then(myEncouragement =>
           log(`=> user receives the encouragement: ${myEncouragement}`),
         );
-      return 'Thanks for the setup. I sure hope I get some encouragement...\nuser vat is happy\n';
+      return 'Thanks for the setup. I sure hope I get some encouragement...';
```

Comment:
This change will break the swingset-runner tests, which I don't see changed in this PR, and indeed the CI run seems to show them not working.

Reply:
Yes, I admit to leaning on the CI to discover which tests (if any) would be affected. Should be fixed now.

Reply:
Yah.  I always make it a point to do a `yarn test` in any package containing a file I've changed, and only fall back on CI for finding issues that are due to more distant dependencies.

Reply:
In any case, (1) as I said above (mistakenly thinking I was looking at a `swingset-runner` file), you can't really use `vatPowers.testLog` in `swingset-runner` demos, and (2) the "user vat is happy" and "bot vat is happy" text are part of the `swingset-runner` tests and need to go back in.